packages/office-ui-fabric-react/src/components/Button/ButtonBase/ButtonBase.tsx

```diff
@@ -0,0 +1,87 @@
+/* tslint:disable:no-unused-variable no-unused-imports */
+import * as React from 'react';
+/* tslint:enable:no-unused-variable no-unused-imports*/
+import {
+  BaseComponent,
+  css,
+  assign,
+  getId,
+  getNativeProps,
+  buttonProperties,
+  anchorProperties
+} from '../../../Utilities';
+import { IButtonProps, IButton } from './ButtonBase.Props';
+import './ButtonBase.scss';
```

Comment:
Are you talking about the styles? Should I add ButtonBase.scss to each of the individual buttons? The ButtonBase styles are the bare essentials. No default colors.